src/components/side-panel/layer-panel/color-range-selector.js

```diff
@@ -188,6 +227,10 @@ const StyledColorRange = styled.div`
   }
 `;
 
+
```

Comment:
too many new lines

Reply:
re-run formatting and resolved spacing issues